consensus/module_consensus_state_sync.go

```diff
@@ -22,20 +23,69 @@ func (m *consensusModule) GetNodeAddress() string {
 	return m.nodeAddress
 }
 
-// TODO(#352): Implement this function, currently a placeholder.
-// commitReceivedBlocks commits the blocks received from the blocksReceived channel
-// it is intended to be run as a background process
+// blockApplicationLoop commits the blocks received from the blocksResponsesReceived channel in state sync processes
 func (m *consensusModule) blockApplicationLoop() {
```

Comment:
Can you add a subcontexted logger here: https://github.com/pokt-network/pocket/wiki/Logger-Readme#submodule--subcontext-logging